### ISSUE TITLE
Avoid generating duplicate routes?

### ISSUE THREAD

**Issue by jensb** [enriclluelles/route_translator#87]
When using `host_locales`, many routes get duplicated, e.g. I have 

```
 get '/blah' => 'api/v1/foo#bar', as: :blah_foo
```

in a `localized` block in `routes.rb`. This results in (with `available_locales = :de, :en`)

```
    blah_foo_de GET    /de/blah(.:format)      api/v1/foo#bar {:locale=>"de"}
    blah_foo_native_en GET    /blah(.:format)         api/v1/foo#bar {:locale=>"en"}
    blah_foo_native_de GET    /blah(.:format)         api/v1/foo#bar {:locale=>"de"}
    blah_foo_en GET    /blah(.:format)         api/v1/foo#bar {:locale=>"en"}
```

If both `/de/blah` and `/blah` are accessible under the German domain with the same content, this will result in duplicate content penalties in search engines. Can I drop the "native" routes?

**Comment by tagliala** [comment]
This should be possible via `config.force_locale = true`

**Comment by lime** [comment]
> This should be possible via `config.force_locale = true`

However, the README states:

> If host_locales option is set, the following options will be forced (even if you set to true):
> 
>     @config.generate_unlocalized_routes         = false
>     @config.generate_unnamed_unlocalized_routes = false
>     @config.force_locale                        = false
>     @config.hide_locale                         = false

What's the reasoning behind this? Would being able to set `force_locale = true` cause any adverse effects?

**Comment by tagliala** [comment]
@lime you are right, because I was not and I'm not using this solution for the very same reason.

This is my approach:

`route_translator.rb`
```rb
RouteTranslator.config do |config|
  config.hide_locale = true
end
```

`application_controller.rb`
```
  DOMAIN_LANGUAGE = {
    'it.lvh.me' => :it,
    'en.lvh.me' => :en
  }

  skip_around_action :set_locale_from_url
  around_action :set_site_locale

  def set_site_locale(&block)
    I18n.with_locale DOMAIN_LANGUAGE[request.host], &block
  end
```

but I'm having other issues, see: #91, #95

**Comment by tagliala** [comment]
Fixed in the just released 6.0.0.alpha1
